src/main/java/hudson/plugins/jira/JiraChangeLogAnnotator.java

```diff
@@ -26,13 +27,26 @@
  */
 @Extension
 public class JiraChangeLogAnnotator extends ChangeLogAnnotator {
+	
     private static final Logger LOGGER = Logger.getLogger(JiraChangeLogAnnotator.class.getName());
 
+    public JiraChangeLogAnnotator()
```

Comment:
sorry for bothering, but please change same style for brackets everywhere to oracle style :)